conf/messages.en

```diff
@@ -145,18 +143,18 @@ describeYourItem.listItem2 = how it works
 describeYourItem.listItem3 = what it is made from
 describeYourItem.listItem4 = how it is packaged
 
-confidentialInformation.title = Provide information you want to remain confidential
-confidentialInformation.heading = Provide information you want to remain confidential
-confidentialInformation.field1 = Confidential information
+confidentialInformation.title = What information do you want to keep confidential?
+confidentialInformation.heading = What information do you want to keep confidential?
+confidentialInformation.field1 = This information will not be published on the BTI ruling website
```

Comment:
`published on`? or `published to` ?

Reply:
We'll stick with 'published on' I think as that's what users would say